inspirehep/dojson/institutions/fields/bd1xx.py

```diff
@@ -32,20 +36,15 @@
 @utils.filter_values
 def location(self, key, value):
     """GPS location info."""
-    longitude, latitude = ('', '')
-    if value.get('d'):
+    def _get_float(value, c):
         try:
-            longitude = float(value.get('d'))
-        except:
-            pass
-    if value.get('f'):
-        try:
-            latitude = float(value.get('f'))
-        except:
-            pass
+            return float(value[c])
+        except (KeyError, ValueError):
+            return ''
+
     return {
-        'longitude': longitude,
-        'latitude': latitude
+        'longitude': _get_float(value, 'd'),
```

Comment:
Will the code using these fields rely on them being `floats`?

In which case the above:

```python
except (KeyError, ValueError):
             return ''
```

might give some trouble.